content/pages/comparisons/netlify.html

```diff
@@ -0,0 +1,96 @@
+
+  <section>
+    {% call basic.header_lead(
+    header="Read the Docs is the <em><strong>all-in-one</strong></em> solution for your documentation")
+    %}
+    <p>
+      Netlify is fantastic for deploying modern web applications and static sites.
+      However, when it comes to <strong>technical documentation</strong> and <strong>developer-focused content</strong>,
+      Read the Docs is <em>the</em> purpose-built solution.
+    </p>
+    <p>
+      Discover why Read the Docs is the better choice for your documentation projects.
+    </p>
+    {% endcall %}
+  </section>
+
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call comparison.feature_table(other_img="/images/netlify-logo.svg", other_img_alt="Netlify logo") %}
+    {{ comparison.feature_row("Build your site on the cloud", "Generate your site's content on the cloud", "fa-gears", on_other=True) }}
+    {{ comparison.feature_row("Deploy previews", "Solve style issues before deploying", "fa-rocket-launch", on_other=True) }}
+    {{ comparison.feature_row("Static hosting", "Publish any static content", "fa-page", on_other=True) }}
+    {{ comparison.feature_row("Custom 404 pages", "Style 404 pages with context-related information", "fa-xmark", on_other=True) }}
+    {{ comparison.feature_row("Redirects", "Redirect users to the right page", "fa-arrows-turn-right", on_other=True) }}
+    {{ comparison.feature_row("Custom theme", "Style the documentation as you like", "fa-pen-swirl", on_other=True) }}
+    {{ comparison.feature_row("Documentation tool agnostic", "Use the documentation tool your team prefers (Sphinx, MkDocs, Docusaurus, etc)", "fa-book", on_other=True) }}
+    {{ comparison.feature_row("Page analytics", "Understand how people browse your documentation", "fa-chart-mixed", on_other=True) }}
+    {{ comparison.feature_row("Advanced collaboration", "Unlimited users", "fa-people-group", on_other=True) }}
+    {{ comparison.feature_row("Search", "Always find what you are looking for", "fa-magnifying-glass") }}
+    {{ comparison.feature_row("Multiple versions", "Publish multiple versions of your documentation", "fa-books") }}
+    {{ comparison.feature_row("Translations", "Serve translated versions of your docs", "fa-language") }}
+    {{ comparison.feature_row("Visual diff on deploy previews", "Visualize differences directly on the rendered version", "fa-face-monocle") }}
+    {{ comparison.feature_row("Search analytics", "Explore what people search for in your docs", "fa-chart-simple") }}
+    {{ comparison.feature_row("Private documention", "Give access only to trusted people", "fa-lock") }}
+    {{ comparison.feature_row("Unlimited builds per month", "Trigger as many builds as you need", "fa-infinity") }}
+    {{ comparison.feature_row("Open Source", "No lock in and contribute back", "fa-brands fa-osi") }}
+    {% endcall %}
+
+  </div>
+</section>
+
+{# Try it out code block #}
+{% include "includes/try-it-out-mkdocs.html" %}
```

Comment:
I would use something that's more `npm`-based, since it works better when comparing with Netlify. There are more npm-based projects there than Python.